### PR TITLE
Add documentation for ST7789-based displays

### PR DESCRIPTION
**Pull request by agarthetiger**
Hi, I've got an AdaFruit 1.3" TFT Bonnet with Joystick and am learning how to drive the display. I've had a bit of a learning curve with this and wanted to include the ST7789 module to the readthedocs site for this repo so that anyone else googling for information for this display might find these docs a bit quicker than I did :) 

I think I've made the right changes, I bumped the version number and build the docs locally which look fine in a browser on my machine (famous last words!). Apologies if I've missed anything or changed something I shouldn't. I'm happy to make any required modifications to this PR.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -48,9 +48,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = "1.0"
+version = "3.10"
```
Comment (ladyada) [inline docs/conf.py:51]: i think leave these two be (plz revert)

File: docs/conf.py [hunk 2]
```diff
@@ -48,9 +48,9 @@
...
 # The full version, including alpha/beta/rc tags.
-release = "1.0"
+release = "3.10.2"
```
Comment (ladyada) [inline docs/conf.py:53]: i think leave these two be (plz revert)

**Review by ladyada — CHANGES_REQUESTED** [review]

File: docs/conf.py (hunk 1 above)
Comment (agarthetiger) [reply docs/conf.py]: Sure thing, I've reverted these both.

File: docs/conf.py (hunk 2 above)
Comment (agarthetiger) [reply docs/conf.py]: Sure thing, I've reverted these both.

**Review by ladyada — APPROVED** [review]
nice!

**Comment by ladyada** [comment]
looks good! lets see how it goes :)

**Comment by agarthetiger** [comment]
Thank you! :)

**Pull request merged**